website/src/blogs/how-to-deploy-nuodb-on-kubernetes-with-openebs.md

```diff
@@ -0,0 +1,192 @@
+
+    #Use the following YAMLs to create a cStor Storage Pool.
+    # and associated storage class.
+    apiVersion: openebs.io/v1alpha1
+    kind: StoragePoolClaim
+    metadata:
+      name: cstor-disk
+    spec:
+      name: cstor-disk
+      type: disk
+      poolSpec:
+        poolType: striped
+      # NOTE - Appropriate disks need to be fetched using `kubectl get disks`
+      #
+      # `Disk` is a custom resource supported by OpenEBS with `node-disk-manager`
+      # as the disk operator
+    # Replace the following with actual disk CRs from your cluster `kubectl get disks`
+    # Uncomment the below lines after updating the actual disk names.
+      disks:
+        diskList:
+    # Replace the following with actual disk CRs from your cluster from `kubectl get disks`
+    #   - disk-184d99015253054c48c4aa3f17d137b1
+    #   - disk-2f6bced7ba9b2be230ca5138fd0b07f1
+    #   - disk-806d3e77dd2e38f188fdaf9c46020bdc
+    #   - disk-8b6fb58d0c4e0ff3ed74a5183556424d
+    #   - disk-bad1863742ce905e67978d082a721d61
+    #   - disk-d172a48ad8b0fb536b9984609b7ee653
+    ---
+
+### Create the Storage Class
+
+You must configure a StorageClass to provision a cStor volume on a given cStor pool. StorageClass is the interface through which most of the OpenEBS storage policies are defined. I will be using a StorageClass to consume the cStor Pool that is created using external disks attached on the Nodes. Since NuoDB is a statefulset application, it requires only a single storage replica. Therefore, the cStor volume replicaCount is =1. A sample YAML named openebs-nuodb.yaml used to consume cStor pool with cStoveVolume Replica count as 1 is provided in the configuration details below.
+
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: openebs-nuodb
+      annotations:
+        openebs.io/cas-type: cstor
+        cas.openebs.io/config: |
+          - name: StoragePoolClaim
+            value: "cstor-disk"
+          - name: ReplicaCount
+            value: "1"       
+    provisioner: openebs.io/provisioner-iscsi
+    reclaimPolicy: Delete
+    ---
+
+### Import the NuoDB CE Template
+
+Import the NuoDB CE template *ce-template-persistent.yaml* into OpenShift by navigating to the “Overview” Tab, clicking “Import YAML/JSON,” and running the import.
+![Import the NuoDB CE Template](/public/images/blog/import-the-nuodb-ce-template.png) Import YAML Template
+Follow the installation prompts and change the “SM persistent storage class name” and “Admin persistent storage class name” to *openebs-nuodb*.
+![Set Persistent Storage Class Name](/public/images/blog/set-persistent-storage-class-name.png) Set Persistent Storage Class Name
```

Comment:
same here

Reply:
Please add a line space between the text before image

Reply:
Please do the same everywhere where the line space is missing before image